composer.json

```diff
@@ -32,33 +32,32 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/asset": "^4.4 || ^5.4 || ^6.0",
```

Comment:
We can drop the support for `symfony/* ^4.4` as 4.4 is not compatible with PHP 8 anyway.